pkgs/applications/science/math/sage/default.nix

```diff
@@ -46,7 +46,7 @@ let
     # just one 16x16 logo is available
     logo32 = "${sage-src}/src/doc/common/themes/sage/static/sageicon.png";
     logo64 = "${sage-src}/src/doc/common/themes/sage/static/sageicon.png";
-  };
+  } // lib.optionalAttrs withDoc { extraPaths = { "doc" = "${sagedoc}/share/doc/sage/html/en"; }; };
```

Comment:
#179828

Reply:
Thanks! For some reason ofBorg didn't catch this when I reviewed the patch back in March.

Reply:
I guess the reason is that, since Sage 9.6, `sagedoc` uses the `jupyter-kernel-specs` variable (https://github.com/NixOS/nixpkgs/blob/3614186dd69f949bc6a0daa6051fe0ade15a9cb9/pkgs/applications/science/math/sage/sagedoc.nix#L36). Sorry, I should have caught that.